src/neynar-api/v2/openapi-farcaster/apis/user-api.ts

```diff
@@ -593,17 +593,17 @@ export const UserApiAxiosParamCreator = function (configuration?: Configuration)
          * @summary Search for Usernames
          * @param {string} apiKey API key required for authentication.
          * @param {string} q 
-         * @param {number} viewerFid 
+         * @param {number} [viewerFid] 
+         * @param {number} [limit] 
+         * @param {string} [cursor] Pagination cursor.
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        userSearch: async (apiKey: string, q: string, viewerFid: number, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
+        userSearch: async (apiKey: string, q: string, viewerFid?: number, limit?: number, cursor?: string, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
Please reflect these changes in `searchUser` method in `src/neynar-api/v2/client.ts` and `src/neynar-api/neynar-api-client.ts`

also update the JS doc string for those `searchUser`